### ISSUE TITLE
Add --file flag to check and cryptcheck to treat source and destination as single files

### ISSUE THREAD

**Issue by dr-mcgillicuddy** [rclone/rclone#3897]
#### What is your current rclone version (output from `rclone version`)?
rclone v1.50.2
- os/arch: darwin/amd64
- go version: go1.13.4

#### What problem are you are trying to solve?

Allow rclone check and rclone cryptcheck to be run against two files with different names.

rclone check and rclone cryptcheck do not allow both the source and destination paths to be specified as specific files.  Once a file has been renamed on a remote, neither rclone check nor rclone cryptcheck can be run against that file with accurate results--even if the underlying data has not changed, rclone will report an error.

In other words, there is no way to check the integrity of one file on a remote against its local counterpart, unless both files have exactly the same name.

#### How do you think rclone should be changed to solve that?

Add another flag to rclone check and rclone cryptcheck (maybe --file or something like that) to signal that both source and destination should be treated as single files, and that the specified operation should be performed against them, regardless of name.

**Comment by ivandeex** [comment]
@ncw
Probably, a new flag would not be not needed.
`check` should would just compare two files if both `src` and `dest` are files.
Do you see any technical complications?

**Comment by ncw** [comment]
Yes `checkto` seems like a much better idea.

**Comment by ivandeex** [comment]
Will submit a PR
